src/Accounts/Accounts/CommonModule/VTable.cs

```diff
@@ -42,12 +42,16 @@ public class VTable
         /// <summary>
         /// The cmdlet will call this when it is trying to fill in a parameter value that it needs
         /// </summary>
-        /// <param name="resourceId"><c>string</c>containing the expected resource id (ie, ARM).</param>
-        /// <param name="moduleName"><c>string</c>containing the name of the module being loaded.</param>
-        /// <param name="invocationInfo">The <see cref="System.Management.Automation.InvocationInfo" /> from the cmdlet</param>
-        /// <param name="correlationId">The <see cref="string" /> containing the correlation id for the cmdlet</param>
-        /// <param name="name">The <see cref="string" /> parameter name being asked for</param>
-        /// <example>public object GetParameterValue(string resourceId, string moduleName, System.Management.Automation.InvocationInfo invocationInfo, string name)</example>
+        /// <example>
+        /// <code>
+        /// public object GetParameterValue(string resourceId, string moduleName, System.Management.Automation.InvocationInfo invocationInfo, string name)
+        /// </code>
+        /// <para>resourceId: <c>string</c>containing the expected resource id (ie, ARM).</para>
+        /// <para>moduleName: <c>string</c>containing the name of the module being loaded.</para>
+        /// <para>invocationInfo: The <see cref="System.Management.Automation.InvocationInfo" /> from the cmdlet</para>
+        /// <para>correlationId: The <see cref="string" /> containing the correlation id for the cmdlet</para>
+        /// <para>name: The <see cref="string" /> parameter name being asked for</para>
+        /// </example>
         public GetParameterDelegate GetParameterValue;
 
```

Comment:
There are no parameters called resourceId,moduleName, etc. Move them to example block